proc.go

```diff
@@ -256,3 +256,18 @@ func (p Proc) fileDescriptors() ([]string, error) {
 func (p Proc) path(pa ...string) string {
 	return p.fs.Path(append([]string{strconv.Itoa(p.PID)}, pa...)...)
 }
+
+func (p Proc) fileDescriptorInfo(fd string) ([]byte, error) {
```

Comment:
This needs to capitalized to be available from outside of the procfs package.  Also, I think this should return a `FDInfo` with the data already parsed, to make it easier to use.